tests/backends/test_creation.py

```diff
@@ -12,11 +12,17 @@
     DatabaseCreation as MySQLDatabaseCreation
 from django.db.backends.oracle.creation import \
     DatabaseCreation as OracleDatabaseCreation
-from django.db.backends.postgresql.creation import \
-    DatabaseCreation as PostgreSQLDatabaseCreation
 from django.db.utils import DatabaseError
 from django.test import SimpleTestCase, TestCase
 
+try:
```

Comment:
Something like this might reduce the risk of hiding import errors:
```python
try:
    import psycopg2
except ImportError:
    pass
else:
    from psycopg2 import errorcodes
    from django.db.backends.postgresql.creation import \
        DatabaseCreation as PostgreSQLDatabaseCreation
```